src/framework/theme/components/chat/chat-message.component.ts

```diff
@@ -193,15 +202,34 @@ export class NbChatMessageComponent {
    */
   @Input() type: string;
 
+  @Input() customMessageData: any;
```

Comment:
Add comment describing this input. Something like `Data with will be set as custom message template context`.